uber/site_sections/preregistration.py

```diff
@@ -276,9 +276,10 @@ def dealer_confirmation(self, session, id):
 
     def group_members(self, session, id, message=''):
         group = session.group(id)
+        charge = Charge([group, group.leader]) if group.leader else Charge(group, group.leader)
```

Comment:
I think you want the else expression to just be ``Charge(group)``, since otherwise you're passing ``None`` as the second parameter.  This is definitely not what you want to be doing.  As it happens, passing ``None`` as the second parameter has no effect since the second parameter is ``amount``, which will default to calculating the correct amount based on the first ``targets`` parameter if it's ``None``.  But this is still a mistake and the expression should definitely just be ``Charge(group)``.

Once that's fixed, this looks good to go.  Thanks for tackling this; I meant to do it myself last night while addressing those other tickets, but it slipped my mind (probably because we were all too lazy to make an issue for it, haha).